pages/_document.tsx

```diff
@@ -20,9 +20,6 @@ class MyDocument extends Document {
       hasToRemoveScriptWrapper: true,
     });
 
-    newrelic.setTransactionName("App started transaction");
-    newrelic.addCustomAttribute("AppAttribute", "NextJs New Relic attribute");
```

Comment:
These two lines are linked to/mentioned in the public blog post

Reply:
I would update blog post. the Next.js instrumentation names transactions

Reply:
If you keep this all the instrumentation naming gets superseded by this